### PR TITLE
Addded version 0.2.0 and link for 2018/2020 data

### PR DESCRIPTION
**Pull request by pholleway**
Before you submit a pull request, please fill in the following:

**Related Issue(s):**
update version to reflect major change and added links for access to original 2018 and 2020 data 

**PR checklist:**

- [X] Code is formatted (run `scripts/format`).
- [X] Code lints properly (run `scripts/lint`).
- [X] Tests pass (run `scripts/test`).
- [X] Changes are added to the [CHANGELOG](../CHANGELOG.md).
- [X] Examples are added to `examples/`

### CODE REVIEW

File: src/stactools/drcog_lulc/constants.py [hunk 1]
```diff
@@ -103,9 +103,16 @@
...
+        target="https://landcoverarchive.s3.amazonaws.com/2018/lulc_pilot_raster_2018.zip",
+        title="Land_Cover_Raster_Data_2018",
+        media_type="application/zip",
+    ),
+    Link(
+        target="https://landcoverarchive.s3.amazonaws.com/2020/DRCOG_2020_Landcover.zip",
+        title="Land_Cover_Raster_Data_2020",
+        media_type="application/zip",
+    ),
+]
```
Comment (pjhartzell) [inline src/stactools/drcog_lulc/constants.py]: I think two distinct data links would be fine: `DATA_LINK_2018` and `DATA_LINK_2020`. Also, add `rel="original'` to the 2020 link.

Then just add the new links to the `collection.add_links` list in `stac.py`.

Comment (pholleway) [reply src/stactools/drcog_lulc/constants.py]: Thanks, Preston. Added your comments.

**Review by pjhartzell — APPROVED** [review]
Looks good. I'll parse through all my changes and add the relevant ones to the changelog under the 0.2.0 section you created. I'll do that in a new PR, so feel free to merge this.

**Comment by pholleway** [comment]
> Looks good. I'll parse through all my changes and add the relevant ones to the changelog under the 0.2.0 section you created. I'll do that in a new PR, so feel free to merge this.

Sounds good. Thanks!

**Pull request merged**